jjb/core/core-consul.yaml

```diff
@@ -19,18 +19,30 @@
           docker_tag: 'master'
     jobs:
       - '{project-name}-{stream}-verify-docker'
-      - '{project-name}-{stream}-merge-docker'
-      - '{project-name}-{stream}-release-version-docker-daily-no-sonar'
+      - '{project-name}-{stream}-merge-docker':
+          custom_triggers:
+            - reverse:
+                jobs: '{project-name}-{stream}-merge-java'
```

Comment:
See my review comment for context. Not following why this is needed, I thought we were removing merge-java references.